### PR TITLE
Basic Redis resource definition

### PR DESCRIPTION
**Pull request by dharsanb**
https://humanitec.atlassian.net/browse/WAL-8913

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ The following resources are included:
 
 * [mysql/basic](./humanitec-resource-defs/mysql/basic): A basic MySQL.
 * [postgres/basic](./humanitec-resource-defs/postgres/basic): A basic Postgres.
+* [redis/basic](./humanitec-resource-defs/postgres/basic): A basic Redis.
```
Comment (johanneswuerbach) [inline README.md:15]:
```suggestion
* [redis/basic](./humanitec-resource-defs/redis/basic): A basic Redis.
```

File: humanitec-resource-defs/redis/basic/main.tf [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+            securityContext:
+              allowPrivilegeEscalation: false
+              capabilities:
+                drop:
+                  - ALL
+              privileged: false
+              readOnlyRootFilesystem: true
+            image: redis:alpine
+            ports:
+            - containerPort: {{ .init.port }}
```
Comment (johanneswuerbach) [inline humanitec-resource-defs/redis/basic/main.tf:58]: This won't work as redis always runs on `6379` unless changed by config. I think the easiest would be hardcoding `6379` here and modifying the service port mapping below to:

```
      ports:
      - name: tcp-redis
        port: {{ .init.port }}
        targetPort: `6379`
```

This makes the "public" facing port customisable, but keeps the internal port constant.

File: README.md [hunk 3]
```diff
@@ -12,6 +12,7 @@ The following resources are included:
...
 
 The `humanitec-resource-defs` directory includes the respective resource definitions.
```
Comment (johanneswuerbach) [inline README.md:17]: Unrelated, but could you also add an example in the `./examples` folder?

**Review by johanneswuerbach — COMMENTED** [review]
Looks good so far. Left a few comments :)

File: README.md (hunk 3 above)
Comment (dharsanb) [reply README.md:17]: Will create another PR for the same

**Review by johanneswuerbach — APPROVED** [review]

**Pull request merged**